core/common/error.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_ERROR
+#define KAGOME_ERROR
+
+#include "outcome/outcome.hpp"
+
+namespace kagome {
+
+  enum class Error {
+    /// Special code for non error
+    NO_ERROR = 0,
+    /// Common code substitution for some non-important error
+    OTHER_ERROR = 1,
+    /// Common code substitution for some uncaught exception
+    OTHER_EXCEPTION = 2
```

Comment:
To be honest I can't grasp why this can be used given that it bears absolutely no information about the occurred error. If one wants to suppress an error somewhere, they can just log it and return a success, as I understand.

Reply:
Removed